src/components/AvatarWithImagePicker.js

```diff
@@ -125,8 +125,8 @@ class AvatarWithImagePicker extends React.Component {
                                     onItemSelected={() => this.setState({isMenuVisible: false})}
                                     menuItems={this.createMenuItems(openPicker)}
                                     anchorPosition={this.props.anchorPosition}
-                                    animationIn="fadeInRight"
-                                    animationOut="fadeOutRight"
+                                    animationIn="fadeInDown"
+                                    animationOut="fadeOutUp"
```

Comment:
Okay, so I know this is part of your proposal but it's sort of unsettling for my OCD. Because as i test this whenever I click edit, for a brief second the menu is over my avatar which is really weird for me. Not sure if it's just me so I'll let @shawnborton chime in his preference since it's mostly a design thing.

Reply:
I just move the solution to a simpler one. Otherwise, we have to adjust all of our Popups.

Reply:
@shawnborton Could you please provide your feedback? Thanks. What Chirag is saying is that Menu is shown with fade In down effect which covers some part of the avatar while animating to the bottom. Do you feel that this is an UX issue?

Reply:
I agree with this feedback and I've personally felt the same way about the way the popover launches when you tap on the little phone/call button on desktop. That being said, I think it makes sense to just implement the simple solution Rajat has in place for now, but it might be nice to revisit this as a more holistic thing and make sure we launch these popovers starting from the bottom of the element. Or even just a simple fade in would do the trick.

Reply:
Alright, thanks for the thoughts Shawn. I'll go ahead and merge and we can revisit improvements at a later point.